packages/host/src/lib/csi-controller.ts

```diff
@@ -467,8 +457,36 @@ export class CSIController extends TypedEmitter<Events> {
         // operations
         this.router.op("post", "/_monitoring_rate", RunnerMessageCode.MONITORING_RATE, this.communicationHandler);
         this.router.op("post", "/_event", RunnerMessageCode.EVENT, this.communicationHandler);
-        this.router.op("post", "/_stop", RunnerMessageCode.STOP, this.communicationHandler);
-        this.router.op("post", "/_kill", RunnerMessageCode.KILL, this.communicationHandler);
+
+        this.router.op("post", "/_stop", async (req) => {
```

Comment:
@pietrzakacper can you extract this function to a class method?

Reply:
fixed